modules/apps/asset/asset-browser-web/src/main/java/com/liferay/asset/browser/web/internal/display/context/AssetBrowserDisplayContext.java

```diff
@@ -422,6 +422,15 @@ public boolean isShowAssetEntryStatus() {
 	}
 
 	public boolean isShowBreadcrumb() {
+		String scopeGroupType = ParamUtil.getString(
+			_httpServletRequest, "scopeGroupType");
+
+		if (Validator.isNotNull(scopeGroupType) &&
+			scopeGroupType.equals("page")) {
```

Comment:
Can we move this literals to a *Constants class? They're used across multiple modules, so they should be API.